src/NuGet.CommandLine/Commands/DownloadCommandBase.cs

```diff
@@ -78,6 +79,21 @@ protected IEnumerable<Packaging.PackageReference> GetInstalledPackageReferences(
         {
             if (File.Exists(projectConfigFilePath))
             {
+                try
+                {
+                    var xDocument = XDocument.Load(projectConfigFilePath);
+                }
+                catch (XmlException ex)
+                {
+                    var message = string.Format(
+                        CultureInfo.CurrentCulture,
+                        LocalizedResourceManager.GetString("Error_PackagesConfigParseError"),
+                        projectConfigFilePath,
+                        ex.Message);
+
+                    throw new CommandLineException(message);
+                }
+
                 var reader = new PackagesConfigReader(XDocument.Load(projectConfigFilePath));
```

Comment:
Seems we should move line 97 & 98 into the try block. Otherwise, XDocument.Load() will be called twice when the file is well formed, which is the most common case.